common/src/main/java/software/bernie/geckolib/mixin/client/HumanoidArmorLayerMixin.java

```diff
@@ -39,7 +54,7 @@ public abstract class HumanoidArmorLayerMixin<T extends LivingEntity, A extends
     }
 
     @Inject(method = "renderArmorPiece", at = @At("TAIL"))
-    public void geckolib$cleanupPostRender(PoseStack poseStack, MultiBufferSource bufferSource, T entity, EquipmentSlot equipmentSlot, int packedLight, A baseModel, CallbackInfo ci) {
+    private void geckolib$cleanupPostRender(PoseStack poseStack, MultiBufferSource bufferSource, T entity, EquipmentSlot equipmentSlot, int packedLight, A baseModel, CallbackInfo ci) {
         final ItemStack stack = entity.getItemBySlot(equipmentSlot);
```

Comment:
Dose not use the the `@Share` value like the other injection.

Reply:
Fixed